scripts/create_perf_json.py

```diff
@@ -632,6 +640,28 @@ def smi_json(self) -> metric.MetricGroup:
                           threshold=(metric.Event('smi_cycles') > 0.10))
         ])
 
+    @staticmethod
+    def extract_pebs_formula(formula):
+        new_formula = formula
+        # catch X*min/max($PEBS,Y)
+        if MIN_MAX_PEBS.search(formula):
+            for m in re.finditer(MIN_MAX_PEBS, formula):
+                main_event = m.group(1).strip()
+                min_max = m.group(2)
+                pebs = m.group(3)
+                alternative = m.group(4).strip()
+
+                new_string = f' {min_max}( {main_event}*{pebs} , {main_event} * {alternative} ) '
+                new_formula = re.sub(m.re, new_string, new_formula)
+
+        formula_list = re.split(OPS, new_formula)
+        for element in formula_list:
+            element = element.strip()
+            if '$PEBS' in element:
```

Comment:
The only non-min/max case is Lock_Latency. I think also that max is never used, just min.

Reply:
Yes, most of them could be matched by the min/max pattern. The remaining one(s) are replaced in this way.